### ISSUE TITLE
Crash when running app for the first time

### ISSUE THREAD

**Issue by yetanotherlurkeree** [vector-im/riot-android#2513]
I installed the Riot.im app from F-Droid. However, when I try to run the app, the app closes/crashes. All I see is a white screen popping up for a little moment and closing off again. 

I tried giving permissions for the app and clearing the data. Unfortunateyl that all didn't work. Any ideas where I might find what is preventing the app from running? 

I'm using version 0.8.12 on Android 8.0.0 (Oxygen OS).

**Comment by gunnardag** [comment]
I have the same error, Android 8.1.0 (Oxygen OS)
here's the logcat log from  where I tried starting it.

`08-13 09:10:52.942 1416-1683/im.vector.alpha E/cr_VariationsUtils: Failed reading seed file "/data/user/0/im.vector.alpha/app_webview/variations_seed_new": /data/user/0/im.vector.alpha/app_webview/variations_seed_new (No such file or directory)
08-13 09:10:52.942 1416-1683/im.vector.alpha E/cr_VariationsUtils: Failed reading seed file "/data/user/0/im.vector.alpha/app_webview/variations_seed": /data/user/0/im.vector.alpha/app_webview/variations_seed (No such file or directory)
...
08-13 09:10:53.055 1416-1751/im.vector.alpha I/zygote: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasWideColorDisplay retrieved: 0
08-13 09:10:53.066 1416-1751/im.vector.alpha D/libGLESv2: Successfully load libGLESv2_oneplus.so, this=0xf2fbd004
08-13 09:10:53.098 1416-1416/im.vector.alpha V/FA: onActivityCreated
08-13 09:10:53.102 1416-1751/im.vector.alpha W/VideoCapabilities: Unrecognized profile 2130706433 for video/avc
08-13 09:10:53.102 1416-1751/im.vector.alpha W/VideoCapabilities: Unrecognized profile 2130706434 for video/avc
08-13 09:10:53.107 1416-1751/im.vector.alpha W/VideoCapabilities: Unrecognized profile 2130706433 for video/avc
08-13 09:10:53.107 1416-1751/im.vector.alpha W/VideoCapabilities: Unrecognized profile 2130706434 for video/avc
08-13 09:10:53.110 1416-1751/im.vector.alpha W/VideoCapabilities: Unsupported mime video/divx
08-13 09:10:53.112 1416-1751/im.vector.alpha W/VideoCapabilities: Unsupported mime video/divx4
08-13 09:10:53.124 1416-1751/im.vector.alpha W/VideoCapabilities: Unrecognized profile/level 0/3 for video/mpeg2
08-13 09:10:53.126 1416-1751/im.vector.alpha W/VideoCapabilities: Unrecognized profile/level 0/3 for video/mpeg2
08-13 09:10:53.130 1416-1751/im.vector.alpha W/VideoCapabilities: Unsupported mime video/x-ms-wmv
08-13 09:10:53.139 1416-1751/im.vector.alpha W/VideoCapabilities: Unrecognized profile 2130706433 for video/avc
08-13 09:10:53.139 1416-1751/im.vector.alpha W/VideoCapabilities: Unrecognized profile 2130706434 for video/avc
08-13 09:10:53.149 1416-1751/im.vector.alpha I/VideoCapabilities: Unsupported profile 4 for video/mp4v-es
08-13 09:10:53.200 1416-1663/im.vector.alpha V/FA: Checking service availability
08-13 09:10:53.203 1416-1663/im.vector.alpha V/FA: Service available
08-13 09:10:53.203 1416-1663/im.vector.alpha V/FA: Setting useService: true
08-13 09:10:53.203 1416-1663/im.vector.alpha V/FA: Connecting to remote service
...
08-13 09:10:53.414 1416-1416/im.vector.alpha D/NetworkSecurityConfig: No Network Security Config specified, using platform default
08-13 09:10:53.431 1416-1416/im.vector.alpha D/ShortcutBadger: Finding badger
08-13 09:10:53.433 1416-1416/im.vector.alpha D/ShortcutBadger: Returning badger:me.leolin.shortcutbadger.impl.DefaultBadger
08-13 09:10:53.440 1416-1416/im.vector.alpha E/Memory usage: ---------------------------------------------------
08-13 09:10:53.440 1416-1663/im.vector.alpha V/FA: Connection attempt already in progress
08-13 09:10:53.440 1416-1663/im.vector.alpha V/FA: Activity resumed, time: 394159291
08-13 09:10:53.441 1416-1416/im.vector.alpha E/Memory usage: -----------  app resumed with im.vector.activity.LoginActivity@78bb1c9 -----------------
08-13 09:10:53.442 1416-1416/im.vector.alpha E/Memory usage: ---------------------------------------------------
08-13 09:10:53.442 1416-1416/im.vector.alpha E/Memory usage: usedSize   5 MB
08-13 09:10:53.442 1416-1663/im.vector.alpha I/FA: Tag Manager is not found and thus will not be used
08-13 09:10:53.442 1416-1416/im.vector.alpha E/Memory usage: freeSize   3 MB
08-13 09:10:53.443 1416-1416/im.vector.alpha E/Memory usage: totalSize  9 MB
08-13 09:10:53.443 1416-1416/im.vector.alpha E/Memory usage: ---------------------------------------------------
08-13 09:10:53.443 1416-1663/im.vector.alpha D/FA: Logging event (FE): screen_view(_vs), Bundle[{firebase_event_origin(_o)=auto, firebase_screen_class(_sc)=LoginActivity, firebase_screen_id(_si)=-276950766847992018}]
08-13 09:10:53.444 1416-1416/im.vector.alpha E/Memory usage: availMem   3574 MB
08-13 09:10:53.444 1416-1416/im.vector.alpha E/Memory usage: totalMem   7662 MB
08-13 09:10:53.445 1416-1416/im.vector.alpha E/Memory usage: threshold  216 MB
08-13 09:10:53.445 1416-1416/im.vector.alpha E/Memory usage: lowMemory  false
08-13 09:10:53.446 1416-1416/im.vector.alpha E/Memory usage: ---------------------------------------------------
08-13 09:10:53.459 1416-1416/im.vector.alpha V/NAVIGATION: onResume Activity LoginActivity
08-13 09:10:53.523 1416-1663/im.vector.alpha V/FA: Connection attempt already in progress
08-13 09:10:53.526 1416-1663/im.vector.alpha V/FA: Screen exposed for less than 1000 ms. Event not sent. time: 84
08-13 09:10:53.526 1416-1663/im.vector.alpha V/FA: Connection attempt already in progress
08-13 09:10:53.527 1416-1663/im.vector.alpha V/FA: Activity paused, time: 394159345
08-13 09:10:53.527 1416-1663/im.vector.alpha D/FA: Connected to remote service
08-13 09:10:53.528 1416-1663/im.vector.alpha V/FA: Processing queued up service tasks: 5
08-13 09:10:53.535 1416-1685/im.vector.alpha W/cr_CrashFileManager: /data/user/0/im.vector.alpha/cache/WebView/Crash Reports does not exist or is not a directory
08-13 09:10:53.810 1416-1416/im.vector.alpha I/LoginActivity: ## onDestroy(): IN
08-13 09:10:53.998 1416-1662/im.vector.alpha D/FirebaseInstanceId: topic sync succeeded
08-13 09:10:57.497 1416-1767/im.vector.alpha E/Memory usage: ---------------------------------------------------
08-13 09:10:57.502 1416-1767/im.vector.alpha E/Memory usage: -----------  app suspended -----------------
08-13 09:10:57.503 1416-1767/im.vector.alpha E/Memory usage: ---------------------------------------------------
08-13 09:10:57.505 1416-1767/im.vector.alpha E/Memory usage: usedSize   5 MB
08-13 09:10:57.506 1416-1767/im.vector.alpha E/Memory usage: freeSize   1 MB
08-13 09:10:57.507 1416-1767/im.vector.alpha E/Memory usage: totalSize  7 MB
08-13 09:10:57.508 1416-1767/im.vector.alpha E/Memory usage: ---------------------------------------------------
08-13 09:10:58.655 1416-1663/im.vector.alpha V/FA: Inactivity, disconnecting from the service`

**Comment by bmarty** [comment]
Hi @yetanotherlurkeree ,

Thanks for reporting this issue. In @gunnardag logs, I do not see any logged exception.

I do not know how I can help with this issue, any clue?

Benoît

**Comment by gunnardag** [comment]
No, I'm afraid the error lies mostly because of the Oxygen OS, for some reason the activity seems to end itself peacefully each time you open it. 
I don't see any onCreate logs while running the app, which is strange seeing how either of these should appear from the LoginActivity
```
if (null == getIntent()) {
    Log.d(LOG_TAG, "## onCreate(): IN with no intent");
} else {
    Log.d(LOG_TAG, "## onCreate(): IN with flags " + Integer.toHexString(getIntent().getFlags()));
}
```
but when debug stepping through the code, I do access the else part.

I'll mess around with the code some more, since I have an OxygenOS device and see if I can bypass this somehow, my next guess is something goes wrong and stops in the RiotAppCompatActivity onCreate.

This error also lives in the mini Vector app.

Gunnar
